pkg/kubelet/prober/worker.go

```diff
@@ -316,6 +325,19 @@ func (w *worker) doProbe(ctx context.Context) (keepGoing bool) {
 		return true
 	}
 
+	if resultChanged {
+		var eventType string
+		var eventReason string
+		if result == results.Success {
+			eventType = v1.EventTypeNormal
+			eventReason = events.ProbeSuccess
+		} else {
+			eventType = v1.EventTypeWarning
+			eventReason = events.ProbeFailure
+		}
+		RecordContainerEvent(w.recorder, w.pod, &w.container, eventType, eventReason, "%s probe result changed to %s%s", w.probeType, result, getOutputEventString(result, output))
```

Comment:
`"%s probe result changed to %s%s"` what does this string be like? SHould there be a space between two last `%s`?

Reply:
There's intentionally no space here in a similar fashion to how the FakeRecorder handles extra output.

If output is supplied to getOutputEventString the string appended is `: ` + output, otherwise its nothing

The lack of space is so that if there's no output there's no trailing space

Reply:
Also to note across the project I see no other usages of the EventRecorder by ref: https://cs.github.com/kubernetes/kubernetes?q=record.EventRecorder

Reply:
OK, I got it. Maybe call function `getOutputEventString` -> `getPrefixedOutputEventString`. But not required

Reply:
I'll do that 👍🏻